content/pytorch/concepts/tensors/terms/sum/sum.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: '.sum()'
+Description: 'Returns a tensor of a desired size, filled with random values generated from a uniform distribution in the range [0,1).'
+Subjects:
+  - 'AI'
+  - 'Data Science'
+Tags:
+  - 'AI'
+  - 'Deep Learning'
+  - 'Functions'
+  - 'Machine Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+---
+
+In PyTorch, the **`.sum()`** function is used to compute the sum of all elements in the input tensor. This function is part of the `torch` package.
+
+## Syntax
+
+```pseudo
+torch.sum(input, *, dim=None, keepdim=False, out=None)
+```
+
+The `.sum()` function takes one required parameter `input`, which represents the input tensor whose elements are to be summed. It also has four other optional parameters that can be used to customize the output tensor if needed:
+
+- `dim`: Specifies the dimension along which the sum is to be computed. If `dim` is not specified, the sum is computed over all elements of the input tensor.
+- `keepdim`: If set to `True`, the output tensor will have the same number of dimensions as the input tensor.
+- `out`: Stores the output into existing tensor (if there is any).
```

Comment:
Can we add both the syntax here? Like this:

To calculate the sum of all elements, the following syntax is used:

```pseudo
torch.sum(input, *, dtype=None)
```

- `input`: The input tensor.
- `dtype` (optional): The data type of the output tensor.

To calculate the sum along specific dimensions, the following syntax is used:

```pseudo
torch.sum(input, *, dim=None, keepdim=False, out=None)
```

- `input`: The input tensor.
- `dim` (optional): Specifies the dimension(s) along which the sum is computed. If not specified, the sum is computed over all elements.
- `keepdim` (optional): If set to `True`, the output tensor retains dimensions of size one for the reduced dimensions. The default is `False`.
- `out` (optional): A tensor to store the output into. If not provided, a new tensor is created.